packages/edit-site/src/components/sidebar-navigation-screen-pages/index.js

```diff
@@ -28,7 +28,8 @@ const PageItem = ( { postId, ...props } ) => {
 export default function SidebarNavigationScreenPages() {
 	const { records: pages, isResolving: isLoading } = useEntityRecords(
 		'postType',
-		'page'
+		'page',
+		{ status: 'any' }
```

Comment:
Ensures that we show all pages in the list, regardless of status. 

Is this acceptable?

Reply:
Sounds good to me — we'd want to see all pages in this list, regardless of status, I'd imagine?